tests/trx_generator/trx_provider.hpp

```diff
@@ -28,10 +28,29 @@ namespace eosio::testing {
       std::string    _peer_endpoint_type = "p2p";
       std::string    _peer_endpoint      = "127.0.0.1";
       unsigned short _port               = 9876;
+      // Api endpoint not truly used for p2p connections as transactions are streamed directly to p2p endpoint
+      std::string    _api_endpoint       = "/v1/chain/send_transaction2";
 
       std::string to_string() const {
          std::ostringstream ss;
-         ss << "endpoint type: " << _peer_endpoint_type << " peer_endpoint: " << _peer_endpoint << " port: " << _port;
+         ss << "Provider base config endpoint type: " << _peer_endpoint_type << " peer_endpoint: " << _peer_endpoint
+            << " port: " << _port << " api endpoint: " << _api_endpoint;
+         return std::move(ss).str();
+      }
+   };
+
+   struct ackedTrxTraceInfo {
+      bool _valid = false;
+      unsigned int _block_num = 0;
+      unsigned int _cpu_usage_us = 0;
+      unsigned int _net_usage_words = 0;
+      std::string _block_time = "";
+
+      std::string to_string() const {
+         std::ostringstream ss;
+         ss << "Acked Transaction Trace Info "
+            << "valid: " << _valid << " block num: " << _block_num << " cpu usage us: " << _cpu_usage_us
+            << " net usage words: " << _net_usage_words << " block time: " << _block_time;
          return std::move(ss).str();
```

Comment:
I would remove this `std::move` and the one on line 38. They don't do anyting.

Reply:
Addressed: [f9838a1](https://github.com/AntelopeIO/leap/pull/1229/commits/f9838a118b4018c23a0b113543aa17ef0e116412)